Source/Core/VideoCommon/CommandProcessor.cpp

```diff
@@ -87,11 +87,6 @@ void CommandProcessorManager::DoState(PointerWrap& p)
   p.Do(m_cp_status_reg);
   p.Do(m_cp_ctrl_reg);
   p.Do(m_cp_clear_reg);
-  p.Do(m_bbox_left);
-  p.Do(m_bbox_top);
-  p.Do(m_bbox_right);
-  p.Do(m_bbox_bottom);
-  p.Do(m_token_reg);
```

Comment:
This affects save states, make sure to also update the save state version over in `State.cpp`:
https://github.com/dolphin-emu/dolphin/blob/6ea8edd531e0206cdf996b31df526c5da2a6ca96/Source/Core/Core/State.cpp#L101

Reply:
Good catch, thanks!